test/mpi/f08/rma/c2f2cwinf08.f90

```diff
@@ -9,7 +9,7 @@
 !
```

Comment:
Do we have the same issues with F77 and F90 tests? I believe some of these F08 tests were generated using `test/mpi/maint/f77tof90` but then manually maintained simply because we didn't bother to update that script :) That leftover `This file created from test/mpi/f77/rma/c2f2cwinf.f with f77tof90` is a sign of it.

Reply:
We did manually create the tests. At the time, I was under the impression that there was a hard to automate issue with `implicit none`, but looking at the f77tof90, it seems we already have special handling for it. I can look at updating the script to also generate the f08 tests.